app/controllers/GuidanceController.scala

```diff
@@ -45,13 +46,14 @@ class GuidanceController @Inject() (
     standardView: standard_page,
     questionView: question_page,
     inputView: input_page,
+    dateInputView: input_date_page,
     formProvider: SubmittedAnswerFormProvider,
     service: GuidanceService,
     mcc: MessagesControllerComponents
 ) extends FrontendController(mcc)
   with SessionFrontendController {
 
-  val logger = Logger(getClass)
+  val logger: Logger = Logger(getClass)
 
   def getPage(processCode: String, path: String, p: Option[String]): Action[AnyContent] = sessionIdAction.async { implicit request =>
```

Comment:
This method is exceeding the cyclomatic complexity threshold. I'd be tempted to move the code handling a successful data retrieval (lines 62 to 76 to a private method). I have done something similar for the submission code in EG-1137-Exp.

Reply:
will take a look